cmd/kubepkg/cmd/debs.go

```diff
@@ -14,12 +14,13 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
+// nolint: dupl
```

Comment:
Let's try to remove this.
Same with the ones below :point_down:

Reply:
Fixed.